v22.2/create-and-configure-changefeeds.md

```diff
@@ -63,11 +63,18 @@ To create an {{ site.data.products.enterprise }} changefeed:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
-> CREATE CHANGEFEED FOR TABLE table_name, table_name2 INTO '{scheme}://{host}:{port}?{query_parameters}';
+CREATE CHANGEFEED FOR TABLE table_name, table_name2 INTO '{scheme}://{host}:{port}?{query_parameters}';
 ~~~
 
 {% include {{ page.version.version }}/cdc/url-encoding.md %}
 
+If you do not specify a sink or the `format` option, then the changefeed will hang until canceled. If you create a changefeed without a sink but specify the [`format=csv` option](create-changefeed.html#format), your changefeed will run as a [core-style changefeed](changefeed-for.html) sending messages to the SQL session. For example:
```

Comment:
Our 22.2 cockroach client is better behaved and will automatically switch to ndjson format when running a core-style changefeed. If you've specified another format like csv in the command-line, it'll use that instead.

Which, to emphasize, is basically unrelated to the format flag in the `CREATE CHANGEFEED` statement itself. If you tell the command line to format in json form and the changefeed to format in csv form, you'll get a JSON output with CSV nested inside. If you do the reverse, you'll get comma-separated JSON values.

Reply:
Cool, thanks Aaron, when I was testing I didn't quite interpret the changes correctly... I work on adapting this, thanks!